mne/coreg.py

```diff
@@ -946,7 +946,7 @@ def scale_bem(subject_to, bem_name, subject_from=None, scale=None,
     surfs = read_bem_surfaces(src)
     for surf in surfs:
         surf['rr'] *= scale
-        if nn_scale is not None:
+        if nn_scale is not None and len(surf['nn']) > 0:
```

Comment:
But it's misleading when reading the code -- a new reader will wonder under what conditions this is necessary. This should never happen, and if it does, it's a problem elsewhere. You can change it to a `assert len(surf['nn']) > 0` line if you want, it's clearer that it's something that should never happen and is just an internal sanity check.